data/sql_setup/prepare_schedule_e.sql

```diff
@@ -217,7 +217,7 @@ end
 $$ language plpgsql;
 
 drop trigger if exists ofec_sched_e_queue_trigger on fec_vsum_sched_e;
-create trigger ofec_sched_e_queue_trigger before insert or update or delete
+create trigger ofec_sched_e_queue_trigger instead of insert or update or delete
```

Comment:
Yeah I just read up on it a bit and it seems as though it will work as expected if we are assuming that the scripts that the FEC is running are making inserts, updates, and deletes directly on the view ( take a look here for example: https://vibhorkumar.wordpress.com/2011/10/28/instead-of-trigger/).  So that would have to coordinated with the FEC, and sorry if that was what is meant with coordinating with the DBAs, have finally found some time to focus on this.  

Also, if I'm understanding correctly, if they are indeed doing inserts on the view, than it seems as though the insert, delete, etc. has to be ran on the underlying table as well, we aren't doing that so assuming the FEC is, things will be fine there too.